schema/suggested_genes.js

```diff
@@ -2,7 +2,7 @@ import fetch from "../lib/apis/fetch"
 import { GraphQLObjectType, GraphQLList, GraphQLString } from "graphql"
 
 const SuggestedGeneType = new GraphQLObjectType({
-  name: "SuggestedGenes",
+  name: "SuggestedGene",
```

Comment:
@orta a bit confused. Is the name on `SuggestedGeneType` the only thing that needs to change?